src/components/LandingPage/Footer.tsx

```
@@ -1,113 +1,118 @@
-                    </a>
-                </p>
-            </section>
+        <>
+            <footer className="relative flex h-52 items-end justify-between bg-black px-8 pb-10 md:items-center md:px-20 md:pb-0">
+                <section className="flex flex-col gap-1">
+                    <Link href="/lp" className="flex">
+                        <Image src={PEANUT_LOGO} alt="Peanut Logo" width={110} height={40} />
+                    </Link>
+                    <p className="text-xs text-white">
+                        made with love by{' '}
+                        <a className="underline" href="https://squirrellabs.dev/" target="_blank">
```

Biome (2.4.6)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)